src/sandbox.rs

```diff
@@ -37,33 +37,24 @@ pub fn main_sandbox() {
     }
 }
 
-/// Run the sandbox integrated in the task-maker binary, by executing itself with different command
-/// line arguments.
+/// Run the sandbox integrated in the task-maker-tools binary.
 #[derive(Clone, Debug)]
 pub struct SelfExecSandboxRunner {
```

Comment:
Now that it does not really do self-exec it may be renamed to something better, like `ToolsSandboxRunner`?